src/installation/guides/fde.md

```diff
@@ -136,34 +135,28 @@ Copy the RSA keys from the installation medium to the target root directory:
 # cp /var/db/xbps/keys/* /mnt/var/db/xbps/keys/
 ```
 
-Before we enter the chroot to finish up configuration, we do the actual install.
-Do not forget to use the [appropriate repository
-URL](../../xbps/repositories/index.md#the-main-repository) for the type of
-system you wish to install.
+Before entering the chroot to finish up configuration, it is time to perform the actual install.
+Do not forget to use the [appropriate repository URL](../../xbps/repositories/index.md#the-main-repository) 
+for the type of system you wish to install.
+
+Legacy BIOS systems use the `grub` package instead of `grub-x86_64-efi`. For LVM users, The `lvm2` package is required.
 
 ```
-# xbps-install -Sy -R https://repo-default.voidlinux.org/current -r /mnt base-system lvm2 cryptsetup grub
-[*] Updating `https://repo-default.voidlinux.org/current/x86_64-repodata' ...
-x86_64-repodata: 1661KB [avg rate: 2257KB/s]
-130 packages will be downloaded:
-...
+# xbps-install -Sy -R https://repo-default.voidlinux.org/current -r /mnt base-system cryptsetup grub-x86_64-efi
 ```
 
-UEFI systems will have a slightly different package selection. The installation
-command for a UEFI system will be as follows.
+Before entering the chroot, Copy a DNS configuration file from live system to allow internet connection in the chroot.
 
 ```
-# xbps-install -Sy -R https://repo-default.voidlinux.org/current -r /mnt base-system cryptsetup grub-x86_64-efi lvm2
+# cp /etc/resolv.conf /mnt/etc/
 ```
 
-When it's done, we can enter the `chroot` and finish up the configuration.
+Now you can enter the [chroot(1)](https://man.voidlinux.org/chroot.1) and continue configuring the system.
 
 ```
 # chroot /mnt
-# chown root:root /
-# chmod 755 /
 # passwd root
-# echo voidvm > /etc/hostname
+# echo AHOSTNAME > /etc/hostname
```

Comment:
Drop this change, but add the resolv.conf copy before the chroot command.

The chown/chmod are probably superfluous, but it is almost costless to run them and potentially costly should the permissions be wrong.

Reply:
Did you read this part of the comment it opened the PR with:
'Changed example hostname from voidvm to AHOSTNAME to make it more apparent that the user should choose their own hostname.'

Your right, But i've run several installations in QEMU (UEFI and BIOS) and have yet to find a problem with not doing the chown/chmod.